src/js/App.tsx

```diff
@@ -0,0 +1,20 @@
+import { 
+  ChakraProvider,
+} from "@chakra-ui/react";
+import * as React from "react"
+import GlobalSettingsProvider from "./GlobalSettings/GlobalSettingsProvider";
+import MainContainer from "./MainContainer/MainContainer";
+import theme from './Theme/Theme';
+
+
+
+export function App(): JSX.Element {
+
+    return (
+      <ChakraProvider theme = {theme}>
+          <GlobalSettingsProvider>
+              <MainContainer/>
```

Comment:
Any code formatting: tab size 2 and 4, trailing comma exist and doesn't exist, excess empty lines